e2e/davinci-app/main.ts

```diff
@@ -79,9 +97,9 @@ const config: DaVinciConfig = {
 
     const loginBtn = document.getElementById('logoutButton') as HTMLButtonElement;
     loginBtn.addEventListener('click', async () => {
-      await FRUser.logout({ logoutRedirectUri: window.location.href });
+      await FRUser.logout({ logoutRedirectUri: `${window.location.origin}/` });
```

Comment:
because of the above changes , fixed this to use origin so it doesnt include the clientId param